string type
string id
int32 coord_x
int32 coord_y
int32 coord_z
